doc/rados/operations/pools.rst

```diff
@@ -36,8 +36,6 @@ To list your cluster's pools, execute::
 
 	ceph osd lspools
 
-On a freshly installed cluster, only the ``rbd`` pool exists.
-
```

Comment:
Why dropping this ? Could you please clarify ?

Reply:
@joscollin  Currently, the default `rbd` pool is no longer created automatically during cluster creation. Thanks.

Reply:
@Songweibin Alright. My cluster setup (RHEV VMs) had been wiped out recently to verify that. Need to create a new one. Anyway, thanks for the info.